app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_save :update_comments_counter
+
```

Comment:
### SUGGESTION

- [ ] Kindly make the `update_comments_counter` method private 👍🏿